### PR TITLE
Updated Algorithm Chunky Monkey

### PR DESCRIPTION
**Pull request by atjonathan**
Refer #1094

### CODE REVIEW

File: Algorithm-Chunky-Monkey.md [hunk 1]
```diff
@@ -111,7 +114,17 @@ function chunkArrayInGroups(arr, size) {
...
 
-### Solution 2:
+### Code Explanation:
+
+Explain solution here and add any relevant links
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Chunky-Monkey.md]: Remember to add links....

File: Algorithm-Chunky-Monkey.md [hunk 2]
```diff
@@ -129,7 +142,17 @@ function chunkArrayInGroups(arr, size) {
...
 
-### Solution 3:
+### Code Explanation:
+
+Explain solution here and add any relevant links
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Chunky-Monkey.md]: Remember to add links....

**Comment by Rafase282** [comment]
This one needs work, the Read search ask par is missing and the hints part looks like another article in it.

**Comment by Rafase282** [comment]
Please remove the inline code for the links.

**Comment by Rafase282** [comment]
The solutions do not have code explanations.

**Comment by Rafase282** [comment]
Please mark the hints as such and add the relative format that comes with them.

**Comment by atjonathan** [comment]
Working on this now @Rafase282, sorry for the wait :smile:

**Comment by atjonathan** [comment]
@Rafase282, should be ready for merge. Please let me know, if it looks ok and I will squash locally.

**Pull request merged**